aggregator/test/HealthService.test.ts

```diff
@@ -0,0 +1,10 @@
+import { assertEquals } from "./deps.ts";
+
+import Fixture from "./helpers/Fixture.ts";
+
+Fixture.test("HealthService returns healthy", async (fx) => {
+  const healthCheckService = fx.createHealthCheckService()
+  const healthStatus = await healthCheckService.getHealth();
+  const expected = {"status":"HEALTHY"};
```

Comment:
Should be lowercase. Test failure here: https://github.com/web3well/bls-wallet/actions/runs/4593302809/jobs/8111858924?pr=539#step:9:686.